### PR TITLE
feat: Initialize Auth0 with JWT authentication and error handling

### PR DESCRIPTION
**Pull request by KseniiaRiabova**

### CODE REVIEW

**Review by Valeriusdev — APPROVED** [review]

File: server/middleware/jwtMiddleware.js
Comment (NishaVijai) [inline server/middleware/jwtMiddleware.js:1]: This code is new for me to review, since I haven't worked on express and JSON Web Token :-)

**Review by NishaVijai — COMMENTED** [review]
Am I allowed to approve this backend code? I have very little experience in this :-)

File: server/middleware/jwtMiddleware.js
Comment (KseniiaRiabova) [reply server/middleware/jwtMiddleware.js:1]: Thank you for your comment

**Review by NishaVijai — APPROVED** [review]
No virus :-) looks good. Thank you.

**Pull request merged**